VTEX - Payment Provider Protocol.json

```diff
@@ -2702,7 +2702,7 @@
                     },
                     "delayToCancel": {
                         "type": "integer",
-                        "description": "Total time (in seconds) to wait for an authorization and make and automatic call to `/cancellations` to cancel the payment. The minimum value is 600 seconds (10 minutes)."
+                        "description": "Total time (in seconds) to wait for an authorization and make and automatic call to `/cancellations` to cancel the payment. The minimum value is 10 minutes (600 seconds). **NOTE 1**: If you set this field to a time period less than 24 hours (86399 seconds or less), the gateway will retry processing the payment every 1 hour. If you use a value of 24 hours (86400 seconds) or more in this field, the gateway will retry processing the payment every 4 hours. **NOTE 2**:  For `delayToCancel` values between 5 minutes and 1 hour (300 and 3600 seconds), or when payment is made by [PIX](https://developers.vtex.com/docs/guides/payments-integration-pix-instant-payments-in-brazil) (Brazilian instant payment method), retry calls will occur every 5 minutes."
```

Comment:
Typo: "for an authorization and make **an** automatic call" (está "and" automatic call).